ignite/metrics/precision.py

```diff
@@ -9,21 +10,26 @@
 
 class _BasePrecisionRecall(_BaseClassification):
 
-    def __init__(self, output_transform=lambda x: x, average=False):
+    def __init__(self, output_transform=lambda x: x, average=False, is_multilabel=False):
         self._average = average
-        super(_BasePrecisionRecall, self).__init__(output_transform=output_transform)
+        if is_multilabel:
+            if not self._average:
+                warnings.warn("Average argument should be True for multilabel case. "
```

Comment:
Not sure about this warning. Maybe docstring is enough ?

Reply:
Incorporated.